tools/release/release.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  releaseChangelog,
+  releasePublish,
+  releaseVersion,
+} from 'nx/src/command-line/release';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import yargs from 'yargs';
+
+void (async () => {
+  try {
```

Comment:
[Suggestion] We can avoid these wrappings by changing the file extension to `.mts` and adding `.mts` to `tsconfig.eslint.json`.

```suggestion
```

Reply:
I did initially and the lint errors went crazy to I punted on that 
<img width="837" alt="image" src="https://github.com/typescript-eslint/typescript-eslint/assets/900523/839ffa2c-397a-4138-a7a8-eae2b47499b2">

Feel free to adjust in a small follow up if you know what the fix is

Reply:
It's `tsconfig.eslint.json` explicitly including only `.ts` files in this folder, not `.mts`.